i18next-parser.config.js

```diff
@@ -1,6 +1,7 @@
 const fs = require('fs');
 
-const localesEntries = fs.readdirSync('./public/locales');
+// @TODO: make it work with all packages and the web platform
```

Comment:
Should we have a list of TODOs in issues?

Reply:
Or generate a TODO report: https://www.npmjs.com/package/leasot